docs/3_NEW_STACK.md

```diff
@@ -0,0 +1,13 @@
+# 3. Creating a New Stack
```

Comment:
@iphilpot @erikschlegel If this PR is merged, we'll be introducing "Stack" as the official term for Cobalt Templates into the code base. I propose the following alternative: Kobalt Infrastructure Templates (KIT)

Reply:
Updated to use "template" nomenclature for now, as per today's discussion

Reply:
@KeithJRome @erikschlegel @TechnicallyWilliams How about we go with Cobalt Infrastructure Template (CIT) pronounced kit?

Reply:
+1 on CIT